doc/source/whatsnew/v0.22.0.txt

```diff
@@ -330,4 +330,5 @@ Other
 ^^^^^
 
 - Improved error message when attempting to use a Python keyword as an identifier in a ``numexpr`` backed query (:issue:`18221`)
--
+- Updated conda recipe to be in compliance with conda-build 3.0+ (:issue:`18002`)
```

Comment:
I think we now have a section on building changes; don't need the whatsnew about ci